lib/sails-migrations/helpers/config_loader.js

```diff
@@ -51,6 +45,20 @@ function getConfigFromSailsConfig(sailsConfig) {
   return result;
 }
 
+function getConnectionFromSailsConfig(sailsConfig) {
+  if (typeof sailsConfig.defaultAdapter.config.url !== 'undefined' && sailsConfig.defaultAdapter.config.url !== null) {
+    return sailsConfig.defaultAdapter.config.url;
```

Comment:
looks cool, but could you explain a bit more what the purpose of this? why and how this works?

thanks